amplify/backend/function/removeMemberFromColonyWhitelist/src/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "removememberfromcolonywhitelist",
+  "version": "2.0.0",
+  "description": "Lambda function generated by Amplify",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "devDependencies": {
```

Comment:
dropping a comment just in case, all the other lambdas also have `cross-fetch` as the dependency and the `index.js` file has `require('cross-fetch/polyfill');` as the first line. Not sure if it's necessary to include it here too

Reply:
It's only really necessary if you're fetching reputation using `colony-js`